opentech/apply/dashboard/templates/dashboard/applicant_dashboard.html

```diff
@@ -28,7 +28,7 @@ <h6 class="heading heading--no-margin heading--submission-meta"><span>Submitted:
             </div>
             {% include "funds/includes/status_bar.html" with phases=submission.workflow status=submission.phase class="status-bar--small" %}
             {% if request.user|can_edit:submission %}
-                <a href="{% url 'funds:edit_submission' submission.id %}">Edit your submission</a>
```

Comment:
Will we be handling editing the submission at a later point?

Reply:
yes, this will be handled properly next sprint.